run/GCClassic/HISTORY.rc.templates/HISTORY.rc.aerosol

```diff
@@ -332,6 +333,21 @@ COLLECTIONS: 'Restart',
                               'SatDiagnLoss_?LOS?             ',
 ::
 #==============================================================================
+# %%%%% THE SatDiagnEdge COLLECTION %%%%%
+#
+# GEOS-Chem data during satellite overpass
```

Comment:
Add `(on level edges)` here

Reply:
@msulprizio: now fixed in commit d33d1b1